README.md

```diff
@@ -103,12 +103,19 @@ for more details.
       * `mpaso.rst.0002-01-01_00000.nc` (or any other mpas-o restart file)
       * `streams.ocean`
       * `mpas-o_in`
-  * mpas-cice files:
-      * `mpascice.hist.am.timeSeriesStatsMonthly.*.nc`
-      * `mpascice.rst.0002-01-01_00000.nc` (or any other mpas-cice restart
+  * mpas-seaice files:
+      * `mpasseaice.hist.am.timeSeriesStatsMonthly.*.nc`
+      * `mpasseaice.rst.0002-01-01_00000.nc` (or any other mpas-seaice restart
         file)
-      * `streams.cice`
-      * `mpas-cice_in`
+      * `streams.seaice`
+      * `mpas-seaice_in`
+
+Note: for older runs, mpas-seaice files will be named:
+  * `mpascice.hist.am.timeSeriesStatsMonthly.*.nc`
+  * `mpascice.rst.0002-01-01_00000.nc`
+  * `streams.cice`
+  * `mpas-cice_in`
+
```

Comment:
@milenaveneziani, here are the additions that you requested.

Reply:
great thanks.